freeipa/src/main/java/com/sequenceiq/freeipa/controller/ClientTestV1Controller.java

```diff
@@ -19,6 +25,24 @@
     @Inject
     private FreeIpaClientFactory freeIpaClientFactory;
 
+    @Inject
```

Comment:
this class has a bunch of stuff leftover from refactoring/rebasing/squashing. can you remove all the unused stuff? I think you can just revert this class since I don't see any of this being used.